projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -74,20 +77,60 @@ function getSuggestionFromOpenAI(
 		} );
 }
 
+function createPrompt( title = '', content = '', categoryNames = '' ) {
+	// If title is not added, we will only complete.
+	if ( ! title ) {
+		return content;
+	}
+
+	// Title, content and categories
+	if ( content && categoryNames.length ) {
+		return sprintf(
+			/** translators: This will be a prompt to OpenAI to generate a post based on the post title, comma-seperated category names and the last 240 characters of content. */
+			__( "This is a post titled '%1$s', published in categories '%2$s':\n\n … %3$s", 'jetpack' ), // eslint-disable-line @wordpress/i18n-no-collapsible-whitespace
+			title,
+			categoryNames,
+			content
+		);
+	}
+
+	// No content, only title and categories
+	if ( categoryNames.length ) {
+		return sprintf(
+			/** translators: This will be a prompt to OpenAI to generate a post based on the post title, and comma-seperated category names. */
+			__( "This is a post titled '%1$s', published in categories '%2$s':\n", 'jetpack' ), // eslint-disable-line @wordpress/i18n-no-collapsible-whitespace
+			title,
+			categoryNames
+		);
+	}
+
+	// Title and content
+	if ( content ) {
+		return sprintf(
+			/** translators: This will be a prompt to OpenAI to generate a post based on the post title, and the last 240 characters of content. */
+			__( "This is a post titled '%1$s':\n\n…%2$s", 'jetpack' ), // eslint-disable-line @wordpress/i18n-no-collapsible-whitespace
+			title,
+			content
```

Comment:
I think I would remove the content from that translatable string (and above) to avoid that eslint issue. It would also make translation easier imo.

Reply:
I suspect the art of crafting a localized prompt may be much different than a translation. I suspect some languages may require a different whitespace, or no whitespace at all for the same prompt - hence me skewing the rules and including this in translations

Reply:
This whitespace, contrary to other translations does not show on frontend at all, hence I am suggesting this is much different case than usual translation rules